cpp/dolfinx/mesh/utils.h

```diff
@@ -166,18 +166,20 @@ std::vector<std::int32_t> exterior_facet_indices(const Topology& topology);
 ///
 /// @param[in] comm MPI Communicator
 /// @param[in] nparts Number of partitions
-/// @param[in] cell_type Type of cell in mesh
-/// @param[in] cells Cells on this process. The ith entry in list
-/// contains the global indices for the cell vertices. Each cell can
-/// appear only once across all processes. The cell vertex indices are
-/// not necessarily contiguous globally, i.e. the maximum index across
-/// all processes can be greater than the number of vertices. High-order
-/// 'nodes', e.g. mid-side points, should not be included.
+/// @param[in] cell_types List of types of cell in mesh
+/// @param[in] cells List of flattened arrays of cells of each type on this
```

Comment:
What would make this clearest is an example, e.g. "`cells[i]` is a flattened array of cells (shape=`(num_cells, num_cell_vertices`, row-major) of type `cell_types[i]`.